src/StripeTests/BaseStripeTest.cs

```diff
@@ -37,9 +38,11 @@ public BaseStripeTest()
             var initialized = initializer.Value;
 
             // Reset the mock before each test
-            mockHandler.Invocations.Clear();
+            MockHandler.Reset();
         }
 
+        protected static Mock<HttpClientHandler> MockHandler { get => mockHandler; set => mockHandler = value; }
```

Comment:
I needed to expose the mock to subclasses because of advanced mocking needs for the auto-paginator test. Fields should be private, so I wrapped it into a protected property. That's apparently how you're supposed to do this in C# (i.e. StyleCop told me to).